#! /bin/sh /usr/share/dpatch/dpatch-run
## /tmp/name-change.patch.dpatch by  <jari.aalto@cante.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fd
## DP: Author: Jari Aalto <jari.aalto@cante.net>
## DP: 
## DP: 
## DP:     wmShutdown.c: change names

@DPATCH@

diff --git a/wmShutdown.c b/wmShutdown.c
index 7f7e14e..96996bc 100644
--- a/wmShutdown.c
+++ b/wmShutdown.c
@@ -57,7 +57,7 @@ int desliga(void) {
 	FILE *output;
 	char *teste;
 
-	output = popen("/usr/local/bin/Shutdown -h", "r");
+	output = popen("/usr/bin/wmshutdown -h", "r");
 	teste = (char*) fgetc(output);
 	while ((int)teste != EOF) {
 		g_print("%c", teste);
@@ -70,7 +70,7 @@ int reinicia(void) {
 	FILE *output;
 	char *teste;
 
-	output = popen("/usr/local/bin/Shutdown -r", "r");
+	output = popen("/usr/bin/wmhutdown-run -r", "r");
 	teste = (char*) fgetc(output);
 	while ((int)teste != EOF) {
 		g_print("%c", teste);
@@ -148,7 +148,7 @@ int main(int argc, char *argv[]) {
 	gtk_widget_realize(dockArea);	    
 
         icon = (gpointer) gdk_pixmap_create_from_xpm (gtkiw->window, &mask,
-                	NULL, "/usr/share/pixmaps/wmShutdown.xpm");
+                	NULL, "/usr/share/pixmaps/wmshutdown.xpm");
 
         pixmap = gtk_pixmap_new((gpointer) icon, mask);
         gtk_widget_show(pixmap);
